### DISCUSSION TITLE
[4.1.50-scrollbar]是否支持数据量没有达到可滚动状态时不显示滚动条

### DISCUSSION THREAD

**Discussion by YuzKio** [antvis/G2#5572] [Q&A]
如题。一张图表的数据量是不固定的，有时候会触发滚动，如图1.
![image](https://github.com/antvis/G2/assets/75412564/534166b0-90d2-45b8-b775-6052adaa038d)
但有时数据量比较小，如图2，在图2的情况下如何隐藏滚动条
![image](https://github.com/antvis/G2/assets/75412564/a63d3847-4e1d-400d-aecc-a2c87678cff5)

**Comment by pearmini** [comment]
4.x 不支持，5.x 计划支持

**Reply by YuzKio** [reply]
目前4.x中有方法得知溢出状态的改变吗？
